tab_eventos.md

```diff
@@ -7,6 +7,31 @@ tags: Uruguay
 ---
 # Eventos
 
+## 2021
+
+### Jornada de Colaboración - AntiSamy
```

Comment:
I believe here we should say `SAMM`, right?

Reply:
Yes,you're right.